src/data/quot.lean

```diff
@@ -411,6 +411,16 @@ protected def rec_on_subsingleton' {φ : quotient s₁ → Sort*}
   [h : ∀ a, subsingleton (φ ⟦a⟧)] (q : quotient s₁) (f : Π a, φ (quotient.mk' a)) : φ q :=
 quotient.rec_on_subsingleton q f
 
+/-- A version of `quotient.rec_on_subsingleton₂` taking `{s₁ : setoid α} {s₂ : setoid α}`
+as implicit arguments instead of instance arguments. -/
```

Comment:
But `quotient.rec_on_subsingleton₂` doesn't exist, does it?

Reply:
https://leanprover-community.github.io/mathlib_docs/init/data/quot.html#quotient.rec_on_subsingleton%E2%82%82